scripts/generate-mjml-react.ts

```diff
@@ -93,18 +95,21 @@ const ALLOW_ANY_PROPERTY = new Set(
 function getPropTypeFromMjmlAttributeType(
```

Comment:
To go off @IanEdington point about testing, I am thinking we should consider refactoring this script into smaller testable functions. E.g. something like `getPropTypeFromMjmlAttributeType` could be it's own file that is tested